src/runtime/utils/html-tags.ts

```diff
@@ -0,0 +1,119 @@
+export default [
```

Comment:
```suggestion
// Ref https://github.com/sindresorhus/html-tags/blob/v3.2.0/html-tags.json
export default [
```

Do you mind also fixing lints?

Reply:
Sure